gcloud-java-contrib/README.md

```diff
@@ -3,38 +3,10 @@ Google Cloud Java Contributions
 
 Packages that provide higher-level abstraction/functionality for common gcloud-java use cases.
 
-Quickstart
-----------
-If you are using Maven, add this to your pom.xml file
-```xml
-<dependency>
-  <groupId>com.google.gcloud</groupId>
-  <artifactId>gcloud-java-contrib</artifactId>
-  <version>0.1.3</version>
-</dependency>
-```
-If you are using Gradle, add this to your dependencies
-```Groovy
-compile 'com.google.gcloud:gcloud-java-contrib:0.1.3'
-```
-If you are using SBT, add this to your dependencies
-```Scala
-libraryDependencies += "com.google.gcloud" % "gcloud-java-contrib" % "0.1.3"
-```
-
-Java Versions
--------------
-
-Java 7 or above is required for using this client.
-
-Versioning
-----------
-
-This library follows [Semantic Versioning] (http://semver.org/).
-
-It is currently in major version zero (``0.y.z``), which means that anything
-may change at any time and the public API should not be considered
-stable.
+Contents
+--------
+
+ * gcloud-java-nio: NIO Filesystem Provider for Google Cloud Storage.
```

Comment:
Done (though I have no idea how to check the link goes somewhere useful before actually pushing this to github)